hack/test/e2e-qemu.sh

```diff
@@ -160,6 +160,7 @@ function create_cluster {
     --with-init-node=false \
     --cni-bundle-url=${ARTIFACTS}/talosctl-cni-bundle-'${ARCH}'.tar.gz \
     --crashdump \
+    --api-server-balancer-port 13336 \
```

Comment:
like Noel said, this should be structured as optional in one of the pipelines, we don't want to enable not-default settings by default